ui/nuxeo-video/nuxeo-video-conversions.js

```diff
@@ -41,28 +42,20 @@ import '../widgets/nuxeo-tooltip.js';
           a:hover {
             @apply --nuxeo-link-hover;
           }
-          .container {
-            @apply --layout-flex;
-            @apply --layout-horizontal;
-            @apply --layout-center;
-          }
-          .item {
-            @apply --layout-flex;
-          }
         </style>
         <dom-if if="[[label]]">
           <template>
             <h3>[[label]]</h3>
           </template>
         </dom-if>
-        <div>
+        <div class="damProperties">
```

Comment:
what's `.damProperties`?

Reply:
yes good point :/ reverting back

Reply:
it's a class i'm using for all dam cards to be easier to keep the same style - added the css directly in each component now.

Reply:
I believe I understand @nelsonsilva 's question...
"dam properties" may lead to some interpretation, but in fact we are dealing with picture properties. 
And since we saw that we won't move that class for the theme, we may not win a lot by renaming all the the divs from `properties` to `damProperties`

Reply:
well, my idea is to have the same class name in case we need to make changes in the future, but i can revert back the name. the idea of the name being dam was to indicate image and videos.

Reply:
I think I've understood your point too. But all the elements have their own shadow CSS, which may not help on that future vision.
Anyway, I'm really up for consistency! 👍